sms/v1/openapi.yaml

```diff
@@ -81,6 +100,146 @@ paths:
+            per concatenated message.
+            - GSM Unicode alphabet [UCS-2](http://en.wikipedia.org/wiki/GSM_03.38). Maximum length is 70 characters per single message and 66 characters
+            per concatenated message.
+
+            tyntec automatically splits the sent message into concatenated messages if the message
+            exceeds the given limits. These concatenated messages are unified by the handset once again and
+            they are displayed as one message (if supported by the handset).
+
+            `tyntec will charge for each part of the concatenated message as an individual message.`
+
+          required: true
+          schema:
+            type: string
+        - name: encoding
+          in: query
+          description: Encoding selection between GSM7, UNICODE or the default AUTO selection
+          schema:
+            type: string
+            enum:
+              - AUTO
+              - GSM7
+              - UNICODE
+            default: AUTO
+        - name: gateway
+          in: query
+          description: Which gateway to use for the delivery of the message
+          schema:
+            type: string
+            default: DEFAULT
+        - name: conversion
+          in: query
+          description: Is this message subject to conversion ratio
+          schema:
+            type: boolean
+            default: false
+        - name: sendDateTime
+          in: query
+          description: >-
+            Any future date in the format “yyyy-MM-ddT-HH:mm:ss+HH:mm” [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601).
+            If not set, message will be sent immediately.
+            The default time zone is [UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time)
+          schema:
+            type: string
+        - name: validity
+          in: query
+          description: Validity in minutes for this message
+          schema:
+            type: string
+            default: '2880'
+        - name: reference
+          in: query
+          description: A custom reference that will mark the Delivery Report
+          schema:
+            type: string
+        - name: callbackUrl
+          in: query
+          description: Your URL for delivering the Delivery Reports. Leave empty for no Delivery Report
+          schema:
+            type: string
+        - name: callbackMethod
+          in: query
+          description: Your prefered HTTP method for the Delivery Report callback. Possible values POST/GET
+          schema:
+            type: string
+        - name: partLimit
+          in: query
+          description: Up to how many parts you allow this message to be concatenated
+          schema:
+            type: integer
+            format: int32
+            default: 4
+        - name: trate
+          in: query
+          description: >-
+            Up to how much you allow the transcoder to replace original
```

Comment:
Controls the transcoding rate of the original message to [GSM-7](http://en.wikipedia.org/wiki/GSM_03.38) compatible characters. This can be used to compress the message.

Reply:
Done